src/azure-cli/azure/cli/command_modules/appservice/custom.py

```diff
@@ -676,7 +676,7 @@ def enable_zip_deploy_functionapp(cmd, resource_group_name, name, src, build_rem
     return enable_zip_deploy(cmd, resource_group_name, name, src, timeout, slot)
 
 
-def enable_zip_deploy_webapp(cmd, resource_group_name, name, src, timeout=None, slot=None, track_status=False):
+def enable_zip_deploy_webapp(cmd, resource_group_name, name, src, timeout=None, slot=None, track_status=True):
```

Comment:
I would like to confirm if changing this default value will cause a breaking change in the customer's automation workflow?

Reply:
@zhoxing-ms Yes, this will be a breaking change.